src/halmos/warnings.py

```diff
@@ -28,4 +81,4 @@ def url(self) -> str:
 
 
 def warn_code(error_code: ErrorCode, msg: str):
-    logger.warning(f"{color_warn(msg)}\n(see {error_code.url()})")
+    logger.warning(f"{msg}\n(see {error_code.url()})")
```

Comment:
this is to avoid circular dependency. note that now warnings are clearly marked with a level column and indentation, thanks to the rich handler.

Reply:
e.g:
<img width="807" alt="Screenshot 2024-11-20 at 4 41 52 PM" src="https://github.com/user-attachments/assets/2db80c44-dae4-4f04-882a-f146010d8e5b">

<img width="715" alt="Screenshot 2024-11-20 at 4 41 26 PM" src="https://github.com/user-attachments/assets/92bec080-862f-482e-b313-58ccb319653c">